src/System.Device.Gpio.Tests/Mcp3008Tests.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+using System.Device;
+using System.Device.Pwm;
+using Iot.Device.Adc;
+
+using Xunit;
+
+using static System.Device.Gpio.Tests.SetupHelpers;
```

Comment:
NIT: this makes the code a bit harder to follow. I would just instead call `SetupHelpers.CreateAdc()`

Reply:
How is that different from creating a base class and using derived member or using partial classes we use everywhere in corefx?

Reply:
When just reading the code online, first thing people will look for is a method defined in the class, then they will check if this class has a parent class. Both of those checks will fail to tell you where the method is defined, and the `using static` statement is not something that is very intuitive to a lot of devs, so given that all you are really saving here is typing about 10 chars, I would personally just do the full SetupHelpers.CreateAdc() for readability.